library/classes/html2text.class.php

```diff
@@ -34,30 +34,30 @@
 /**
  *  Takes HTML and converts it to formatted, plain text.
  *
- *  Thanks to Alexander Krug (http://www.krugar.de/) to pointing out and
+ *  Thanks to Alexander Krug (https://www.krugar.de/) to pointing out and
  *  correcting an error in the regexp search array. Fixed 7/30/03.
  *
  *  Updated set_html() function's file reading mechanism, 9/25/03.
  *
- *  Thanks to Joss Sanglier (http://www.dancingbear.co.uk/) for adding
+ *  Thanks to Joss Sanglier (https://www.dancingbear.co.uk/) for adding
  *  several more HTML entity codes to the $search and $replace arrays.
  *  Updated 11/7/03.
  *
- *  Thanks to Darius Kasperavicius (http://www.dar.dar.lt/) for
+ *  Thanks to Darius Kasperavicius (https://www.dar.dar.lt/) for
  *  suggesting the addition of $allowed_tags and its supporting function
  *  (which I slightly modified). Updated 3/12/04.
  *
  *  Thanks to Justin Dearing for pointing out that a replacement for the
  *  <TH> tag was missing, and suggesting an appropriate fix.
  *  Updated 8/25/04.
  *
- *  Thanks to Mathieu Collas (http://www.myefarm.com/) for finding a
+ *  Thanks to Mathieu Collas (https://www.myefarm.com/) for finding a
```

Comment:
This link only works with http , so needs to stay as http

Reply:
👍